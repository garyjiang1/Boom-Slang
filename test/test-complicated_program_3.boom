int x = 5
def myfunc(int x, float foo) returns void:
	int y = 5
	int z = 7
	if x > 5:
		if x > 10:
			int z = 20
			println(z)
		elif x > 20:
			println("hey")
		elif x > 30:
			println("hey")
			println("hey")
		else:
			println("hey")
			println(z)

myfunc(15,20.83)
myfunc(7,20.76)
