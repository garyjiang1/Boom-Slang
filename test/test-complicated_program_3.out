20
hey
7
